<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShouYinW</class>
 <widget class="QWidget" name="ShouYinW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton1">
         <property name="text">
          <string>编码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton2">
         <property name="text">
          <string>解码</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox1">
         <property name="text">
          <string>编码1(加密数据的unicode不大于16bit)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox1_2">
         <property name="text">
          <string>编码2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox3">
         <property name="text">
          <string>编码3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>密令(4个字):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>嗷呜啊~</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditIn"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditOut"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
